tests/buildTools/src/test/scala/tests/BaseBuildToolSuite.scala

```diff
@@ -49,8 +50,15 @@ abstract class BaseBuildToolSuite extends MopedSuite(ScipJava.app) {
   // NOTE(olafur): workaround for https://github.com/scalameta/moped/issues/18
   override val temporaryDirectory: DirectoryFixture =
     new DirectoryFixture {
-      private val path = BuildInfo.temporaryDirectory.toPath
-      override def apply(): Path = path
+      private val path = BuildInfo
+        .temporaryDirectory
+        .toPath
+        .resolve(self.getClass().getSimpleName())
```

Comment:
Doing this so that individual suites don't clobber each other (otherwise you get issues with parallel folder deletion and all that).

Hacky but works for now.